docs/user/science.rst

```diff
@@ -0,0 +1,142 @@
+
+    Discuss changes via Pull Request and track all changes in your project's version history.
+
+    Does this sound chaotic? Don't worry, the full ownership and permission handling remains in your hands: Project and organization owners govern what is released and who has access to approve and build changes.
+
+.. dropdown:: 🔎 Search Engine Optimization
+
+    Don't reinvent SEO: Have SEO handled jointly by the combined best-practices from Sphinx, its themes and Read the Docs' platform.
+
+.. dropdown:: 🌱 Grow your own solutions
+
+    The eco-system is Open Source and makes it accessible for anyone with Python skills to build their own extensions.
+
+We want science communities to use Read the Docs and to be part of the documentation community 💞
+
+Getting started: Jupyter Book
+-----------------------------
+
+:external+jupyterbook:doc:`Jupyter Book <intro>` on Read the Docs brings you the rich experience of computated `Jupyter <https://jupyter.org/>`__ documents built together with a modern documentation tool. The results are beautiful and automatically deployed websites, built with Sphinx and :doc:`Executable Book <executablebook:index>` + all the extensions available in this ecosystem.
+
+Here are some popular activities that are well-supported by Jupyter Book:
+
+* Publications and books
+* Course and research websites
+* Interactive classroom activities
+* Data science software documentation
+
+
+.. button-ref:: executablebook:gallery
+    :ref-type: doc
+    :color: primary
+
+    Visit the gallery of solutions built with Jupyter Book
+
+Ready to get started?
+"""""""""""""""""""""
+
+.. Note that this is a deliberate repitition of a previous segment. Should it repeat? Maybe not, but for now it's nice to be sure that people see it.
+
+* All new to this? Take the official :external+jupyterbook:doc:`Jupyter Book Tutorial <start/your-first-book>`
+* Curious for practical code? See :doc:`/examples`
+* Familiar with Sphinx? See :doc:`/guides/jupyter`
+
+
+Examples and users
+""""""""""""""""""
+
+.. TODO: get the correct link for
+.. :external+jupyter:ref:`the many sub-projects of Jupyter <index.md#sub-project-documentation>`
+
+Read the Docs community for science is already big and keeps growing. The :external+jupyter:doc:`Jupyter Project <index>` itself and `the many sub-projects of Jupyter <https://docs.jupyter.org/en/latest/#sub-project-documentation>`__ are built and published with Read the Docs.
+
+.. grid:: 3
+    :gutter: 2
+    :padding: 0
+
+    .. grid-item-card:: Jupyter Project Documentation
+      :img-top: img/logo_jupyter.png
+      :link: https://docs.jupyter.org/
+
+    .. grid-item-card:: Chainladder - Property and Casualty Loss Reserving in Python
+      :img-top: img/logo_chain_ladder.png
+      :link: https://chainladder-python.readthedocs.io/
+
+    .. grid-item-card:: Feature-engine - A Python library for Feature Engineering and Selection
+      :img-top: img/logo_feature_engine.png
```

Comment:
These images could be resized to save some bandwidth when loading the page.

Reply:
It was possible to reduce the Jupyter logo from 117K to 33K.

The other logos aren't possible to reduce in file size by downsampling the image.